[2023-05-06T18:56:27.635+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Summer_Warped_ETL.GetSongsFromPlaylist manual__2023-05-06T03:40:12.069128+00:00 [queued]>
[2023-05-06T18:56:27.640+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Summer_Warped_ETL.GetSongsFromPlaylist manual__2023-05-06T03:40:12.069128+00:00 [queued]>
[2023-05-06T18:56:27.641+0000] {taskinstance.py:1331} INFO - Starting attempt 31 of 33
[2023-05-06T18:56:27.648+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): GetSongsFromPlaylist> on 2023-05-06 03:40:12.069128+00:00
[2023-05-06T18:56:27.655+0000] {standard_task_runner.py:57} INFO - Started process 2794 to run task
[2023-05-06T18:56:27.659+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'Summer_Warped_ETL', 'GetSongsFromPlaylist', 'manual__2023-05-06T03:40:12.069128+00:00', '--job-id', '179', '--raw', '--subdir', 'DAGS_FOLDER/warped-dag.py', '--cfg-path', '/tmp/tmpasprqt1n']
[2023-05-06T18:56:27.660+0000] {standard_task_runner.py:85} INFO - Job 179: Subtask GetSongsFromPlaylist
[2023-05-06T18:56:27.703+0000] {task_command.py:410} INFO - Running <TaskInstance: Summer_Warped_ETL.GetSongsFromPlaylist manual__2023-05-06T03:40:12.069128+00:00 [running]> on host 49f85d1317d0
[2023-05-06T18:56:27.760+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Ananta Moharana' AIRFLOW_CTX_DAG_ID='Summer_Warped_ETL' AIRFLOW_CTX_TASK_ID='GetSongsFromPlaylist' AIRFLOW_CTX_EXECUTION_DATE='2023-05-06T03:40:12.069128+00:00' AIRFLOW_CTX_TRY_NUMBER='31' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-05-06T03:40:12.069128+00:00'
[2023-05-06T18:56:28.216+0000] {logging_mixin.py:149} INFO - howdy
[2023-05-06T18:56:28.218+0000] {taskinstance.py:1847} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 181, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 198, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/src/extract.py", line 62, in get_songs_from_playlist
    for i,item in enumerate(len(r['items'])):
TypeError: 'int' object is not iterable
[2023-05-06T18:56:28.235+0000] {taskinstance.py:1373} INFO - Marking task as UP_FOR_RETRY. dag_id=Summer_Warped_ETL, task_id=GetSongsFromPlaylist, execution_date=20230506T034012, start_date=20230506T185627, end_date=20230506T185628
[2023-05-06T18:56:28.245+0000] {standard_task_runner.py:109} ERROR - Failed to execute job 179 for task GetSongsFromPlaylist ('int' object is not iterable; 2794)
[2023-05-06T18:56:28.297+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 1
[2023-05-06T18:56:28.318+0000] {taskinstance.py:2674} INFO - 0 downstream tasks scheduled from follow-on schedule check
